templates/production.ts

```diff
@@ -0,0 +1,368 @@
+export const PRODUCTION_ACCESS_CONSUMER_TEMPLATE = `
+<head>
+  <style>
+    .card {
+      margin: 0 0 20px 0;
+      background: #FFFFFF;
+      Padding: 24px 24px 0.1px 24px;
+      font-family: Helvetica, Arial, sans-serif;
+      font-size: 16px;
+      line-height: 25px;
+      max-width: 700px;
+    }
+
+    .header {
+      background: #112E51;
+      width: 100%;
+      height: 80px;
+    }
+
+    .header img {
+      margin-top: 30px;
+      margin-left: 20px;
+    }
+
+    .steps {
+      width: 24px;
+      float:left;
+      height: 26px;
+      margin-right: 10px;
+    }
+
+    .steps-container {
+      height: 26px;
+      margin-bottom: 20px;
+    }
+
+    .sub {
+      margin-top: -25px;
+      margin-left: 34px;
+      float:center;
+    }
+
+    h1 {
+      margin-bottom: 20px;
+      padding: 0;
+      font-size: 24px;
+      line-height: 30px;
+      font-weight: bold;
+      color: #323A45;
+    }
+
+    h2 {
+      margin-bottom: 15px;
+      padding: 0;
+      font-size: 16px;
+      line-height: 25px;
+      font-weight: bold;
+      color: #323A45;
+      display: inline;
+    }
+
+    p {
+      margin-bottom: 20px;
+      font-size: 16px;
+      line-height: 25px;
+      color: #323A45;
+    }
+  </style>
+</head>
+<div class="card">
+  <div class="header"><img src="https://dvp-developer-portal-backend-images.s3-us-gov-west-1.amazonaws.com/VA-Lighhouse-APIs-email.png" alt="logo"/></div>
+  <p>We’ve received your request for production access. It’s good to remember that getting production access requires
+    multiple steps and can take less than a week to over a month, depending on the API. For open data APIs,
+    production access takes a week or less and no demo is required.</p>
+  <h1><strong>What’s next?</strong></h1>
+  <div class="steps-container">
+    <img class="steps" src="https://dvp-developer-portal-backend-images.s3-us-gov-west-1.amazonaws.com/Done.png" alt="Step one, done"/><h2>Start developing.</h2>
```

Comment:
these probably shouldn't be h2's because they're not really covering a section per se. also could maybe wrap these in lists rather than divs, since it is semantically a list

Reply:
Aren't they each covering a section? They are individual sections of the `What's Next?` part of the template.

Reply:
I would add the list structure but I would keep the divs. I would also just use the strong tag to apply the bold effect and replace the heading with <p> elements. Example:
<ul>
<li>
<div class="steps-container">
    <img class="steps" src="..." alt="Step two, done"/>
    <p><strong>Submit your production access request.</strong></p>
  </div>
  <p class="sub">
    We’ll review your information and notify you within 1-2 weeks if there are any changes we need.
  </p>
</li>
</ul>

The nice thing about the li is that it groups the step description and its details.

Reply:
These have been changed to use a list in the latest commit!